usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -90,10 +91,14 @@ def account_download_filter(account_type, download_table, filters, account_level
                 "treasury_account__tas_rendering_label",
                 "program_activity__program_activity_code",
                 "object_class__object_class",
+                "object_class__direct_reimbursable",
             ],
         }
         distinct_cols = unique_columns_mapping[account_type]
-        order_by_cols = distinct_cols + ["-reporting_period_start"]
+        order_by_cols = distinct_cols + [
+            "-reporting_period_start",
+            "-pk",
```

Comment:
Does this ensure the results are more correct or is it simply to be deterministic?

If this is valuable, does the pk make it fragile to reloads like we discovered with the FPDS loader?

Reply:
This is purely to make it deterministic.  I don't know that there's a "correct" order if all of the distinct columns are the same.  The non-determinism I observed was eliminated when we added Direct/Reimbursable flag so this is just a bit of defensive programming.